junit-plugin/src/main/scala/org/scalajs/junit/plugin/ScalaJSJUnitPlugin.scala

```diff
@@ -139,8 +141,8 @@ class ScalaJSJUnitPlugin(val global: Global) extends NscPlugin {
       }
 
       private def genConstructor(owner: ClassSymbol): DefDef = {
-        val rhs = gen.mkMethodCall(
-            Super(owner, tpnme.EMPTY), ObjectClass.primaryConstructor, Nil, Nil)
+        val rhs = Block(gen.mkMethodCall(
+            Super(owner, tpnme.EMPTY), ObjectClass.primaryConstructor, Nil, Nil))
```

Comment:
Is the non-porting of the comment here intentional? (We can leave it like this if it is too much git-foo to fix or if it is intentional of course).

Reply:
Oh no that was not intentional. I'll add it. It's easy in this case because it's in the top-most commit, so it's just an amend away.